Trace/Hardware/Device/DeviceModelTests.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DeviceModelTests.swift
+//  Tests
+//
+//  Created by Mukund Agarwal on 23/07/2020.
+//  Copyright © 2020 Bitrise. All rights reserved.
+//
+
+import XCTest
+@testable import Trace
+
+class DeviceModelTests: XCTestCase {
```

Comment:
This file needs to go in the test folder

Reply:
Will fix it. Not sure why Xcode didn't automatically put it in the right place.